src/amo/components/Header/index.js

```diff
@@ -19,26 +19,18 @@ export class HeaderBase extends React.Component {
     i18n: PropTypes.object.isRequired,
     isHomePage: PropTypes.bool.isRequired,
     location: PropTypes.object.isRequired,
-    SearchFormComponent: PropTypes.node.isRequired,
     query: PropTypes.string,
   }
 
   static defaultPropTypes = {
-    SearchFormComponent: SearchForm,
     isHomePage: false,
   }
 
   render() {
-    const {
-      SearchFormComponent,
-      i18n,
-      isHomePage,
-      location,
-      query,
-    } = this.props;
+    const { i18n, isHomePage, location, query } = this.props;
     const headerLink = (
       <Link className="Header-title" to="/">
-        <Icon className="Header-addons-icon" name="fox-light" />
+        <Icon className="Header-addons-icon" name="fox" />
         {
           // translators: "Firefox" should not be translated. :-)
```

Comment:
I know this is old code but does `translators:` work? I thought it had to be `L10n:`.

Reply:
I like *translators* better, just curious :)

Reply:
I believe we added it as an extra keyword. I want to find the code but I forget the repo 😒

Reply:
`translators` is correct https://github.com/mozilla/babel-gettext-extractor/blob/c5361b4ca4df2a484913486d13eb32a879dabf3f/index.js#L27

Reply:
See also https://github.com/mozilla/babel-gettext-extractor/issues/6